test/SILGen/shared.swift

```diff
@@ -1,4 +1,153 @@
 // RUN: %target-swift-frontend -Xllvm -sil-full-demangle -emit-silgen -sil-serialize-witness-tables %s -disable-objc-attr-requires-foundation-module | %FileCheck %s
```

Comment:
1) I don't need to use them, it was just something quick.
2) Enabling the verifier works flawlessly.  I'll update this test to include its output instead.